pkg/initialize/client.go

```diff
@@ -0,0 +1,40 @@
+package initialize
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2018-02-01/storage"
+	azstorage "github.com/Azure/azure-sdk-for-go/storage"
+	"github.com/Azure/go-autorest/autorest/azure/auth"
+
+	"github.com/openshift/openshift-azure/pkg/api"
+)
+
+type azureStorageClient struct {
+	accounts storage.AccountsClient
+	storage  azstorage.Client
+}
+
+func newAzureClients(ctx context.Context, cs *api.OpenShiftManagedCluster) (*azureStorageClient, error) {
+	// TODO: needs to come from property bag
+	authorizer, err := auth.NewAuthorizerFromEnvironment()
```

Comment:
I imagine this will change to use credentials from `Context` ?

Reply:
that's what the comment above is meant to mean ;)

Reply:
#243